README.md

```diff
@@ -287,3 +287,9 @@ Returns valid [`ResponseType`](#schemas)s, eg:
 
 Please log tickets and issues at our [Trapperkeeper Issue Tracker](https://tickets.puppetlabs.com/browse/TK).
 In addition there is a #trapperkeeper channel on Freenode.
+
+## Maintainence
+
+Maintainers: Justin Stoller <justin@puppet.com>, Kevin Corcoran <kevin.corcoran@puppet.com>, Nathaniel Smith <nathaniel@puppet.com>
+
+Tickets: https://tickets.puppetlabs.com/browse/TK
```

Comment:
This is a duplication of the info in the above **Support** section. One of them should be removed, and the remaining content from both makes sense to be merged, IMO.